pkg/3scale/amp/component/system_options_builder.go

```diff
@@ -8,11 +8,7 @@ import (
 )
 
 type S3FileStorageOptions struct {
-	AWSAccessKeyId       string
-	AWSSecretAccessKey   string
-	AWSRegion            string
-	AWSBucket            string
-	AWSCredentialsSecret string
+	S3ConfigSecret string
```

Comment:
We expect the secret name as the S3 file storage option now. At this moment we only care about that. We are not performing any kind of validation of the keys of the secret nor the contents of the keys. We just mount the secret(and automatically all of the keys). What are your thoughts on this?